client/index.jsx

```diff
@@ -18,19 +18,26 @@ grid-row-gap: 3px;
 grid-column-gap: 2px;
 `;
 
+const Container = styled.div`
+position: relative;
+overflow: auto;
+`;
+
 const OpenButton = styled.span`
 z-index: 1;
-top: 12vw;
+top: 45%;
 right: 5%;
 position: absolute;
 text-align: center;
+font-family: Arial, Helvetica, sans-serif;
```

Comment:
Can we all use the same font family please? 🙂
font-family: Roboto, "Helvetica Neue", sans-serif;